src/optimum/rbln/transformers/models/decoderonly/modeling_decoderonly.py

```diff
@@ -651,6 +668,77 @@ def align_2MB(x: int) -> int:
 
         return max_n_blocks
 
+    @classmethod
+    def get_input_info(
+        cls,
+        batch_size: int,
+        query_length: int,
+        use_inputs_embeds: bool,
+        use_attention_mask: bool,
+        max_seq_len: int,
+        kvcache_block_size: int,
+        kvcache_num_blocks: int,
+        model_config: PretrainedConfig,
```

Comment:
how about not using model_config in this function

```suggestion
        head_dim:int,
        num_key_value_heads:int,
```